.vsts.pipelines/builds/matrix.yml

```diff
@@ -29,6 +29,17 @@ jobs:
     matrix:
       Offline: { type: Offline}
 
+- template: ../jobs/ci-linux.yml
+  parameters:
+    job: centos8
+    imageName: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-8-daa5116-20200325130212
+    reportPrebuiltLeaks: false
```

Comment:
This param gets its default here:

https://github.com/dotnet/source-build/blob/9678feff10e08406d11aee34ff33d409de915902/.vsts.pipelines/jobs/ci-linux.yml#L8

is changed from a template param to job variable here:

https://github.com/dotnet/source-build/blob/9678feff10e08406d11aee34ff33d409de915902/.vsts.pipelines/jobs/ci-linux.yml#L34

used as a job variable here:

https://github.com/dotnet/source-build/blob/9678feff10e08406d11aee34ff33d409de915902/.vsts.pipelines/jobs/ci-linux.yml#L117-L119

`/p:UseSystemLibraries=false` could be wired in the same way.

Reply:
Err, I'm pointing to the wrong step there (tarball creation), but would make sense to me to add a similar construct to the production build:

https://github.com/dotnet/source-build/blob/9678feff10e08406d11aee34ff33d409de915902/.vsts.pipelines/jobs/ci-linux.yml#L63-L72

Maybe also the tarball build. 😕

Reply:
Yeah, I've also just been manually editing repos/coreclr.* but adding a build parameter like this is probably the approach we want to take going forward.